docs/tutorials/Raster-data-Lambert93.md

```diff
@@ -59,22 +62,22 @@ const view = new itowns.PlanarView(viewerDiv, viewExtent);
 ```
 
 Several things are done here :
-- First, we retrieve the `DomElement` relative to the container we created to serve as a support for iTowns.
+- First, we retrieve the `DomElement` relative to the container we created that will contain iTowns visualisation.
 - Then we define the geographic extent of our view by instantiating iTowns `{@link Extent}` class.
 To do so, we need to pass five argument to `Extent` constructor, which are :
   - The Coordinates Reference System (CRS) of the extent, given as its EPSG code.
     Here, we use [RGF93 / Lambert-93](https://epsg.io/2154) CRS.
     Not all CRS are defined by default within iTowns, which is why we need to define our CRS using [proj4](https://proj.org/) prior to instantiating the `Extent` ;
   - The westernmost, easternmost, southernmost and northernmost coordinates of the extent.
-- Finally, we create a `PlanarView` giving it the required `DomElement` and `Extent` parameters.
+- Finally, we create a `{@link PlanarView}` giving it the required `DomElement` and `Extent` parameters.
     
 At this stage, our webpage should look like this : 
 
 ![Simple PlanarView](images/Raster-data-Lambert93-1.png)
 
 The blue rectangle you can see is the view we just created.
-By default, iTowns position the camera on top of the view center.
-We can change the initial camera position by passing a `CameraTransformOption` object as an optional parameter of `PlanarView` constructor.
+By default, iTowns positions the camera on top of the view center.
+We can change the initial camera position by passing a [CameraTransformOption]{@link CameraUtils} object as an optional parameter of `PlanarView` constructor.
```

Comment:
Could you add markups around CameraTransformOption as such : [\`CameraTransformOption`] ?

Reply:
yep, done